app/src/androidTest/java/com/android/sample/account/AccountScreenTest.kt

```diff
@@ -45,31 +45,31 @@ class AccountScreenTest {
   private val dummyRecipes: List<Recipe> =
       listOf(
           Recipe(
-              idMeal = "1",
-              strMeal = "Spicy Arrabiata Penne",
-              strCategory = "Vegetarian",
-              strArea = "Italian",
-              strInstructions = "Instructions here...",
+              uid = "1",
+              name = "Spicy Arrabiata Penne",
+              category = "Vegetarian",
+              origin = "Italian",
+              instructions = "Instructions here...",
               strMealThumbUrl =
                   "https://www.recipetineats.com/penne-all-arrabbiata-spicy-tomato-pasta/",
               ingredientsAndMeasurements =
                   listOf(Pair("Penne", "1 pound"), Pair("Olive oil", "1/4 cup"))),
           Recipe(
-              idMeal = "2",
-              strMeal = "Chicken Curry",
-              strCategory = "Non-Vegetarian",
-              strArea = "Indian",
-              strInstructions = "Instructions here...",
+              uid = "2",
+              name = "Chicken Curry",
+              category = "Non-Vegetarian",
+              origin = "Indian",
+              instructions = "Instructions here...",
               strMealThumbUrl =
                   "https://www.foodfashionparty.com/2023/08/05/everyday-chicken-curry/",
               ingredientsAndMeasurements =
                   listOf(Pair("Chicken", "1 pound"), Pair("Curry powder", "2 tbsp"))),
           Recipe(
-              idMeal = "3",
-              strMeal = "Burger with Fries",
-              strCategory = "Fast Food",
-              strArea = "American",
-              strInstructions = "Instructions here...",
+              uid = "3",
+              name = "Burger with Fries",
+              category = "Fast Food",
+              origin = "American",
+              instructions = "Instructions here...",
               strMealThumbUrl =
```

Comment:
Is there a reason for the Mealthumbnail to keep it's str prefix?

Reply:
Hey yeah, it is because we re planning to delete it once we have the fixed structure of our `Recipe`, this parameter will be replaced by the current parameter `url`